src/ledger/LedgerTxnTrustLineSQL.cpp

```diff
@@ -608,41 +425,32 @@ class BulkLoadTrustLinesOperation
         sqlite3_reset(st);
         sqlite3_bind_pointer(st, 1, cstrAccountIDs.data(), "carray", 0);
         sqlite3_bind_int(st, 2, static_cast<int>(cstrAccountIDs.size()));
-        sqlite3_bind_pointer(st, 3, cstrIssuers.data(), "carray", 0);
-        sqlite3_bind_int(st, 4, static_cast<int>(cstrIssuers.size()));
-        sqlite3_bind_pointer(st, 5, cstrAssetCodes.data(), "carray", 0);
-        sqlite3_bind_int(st, 6, static_cast<int>(cstrAssetCodes.size()));
+        sqlite3_bind_pointer(st, 3, cstrAssets.data(), "carray", 0);
+        sqlite3_bind_int(st, 4, static_cast<int>(cstrAssets.size()));
         return executeAndFetch(prep.statement());
     }
 
 #ifdef USE_POSTGRES
     virtual std::vector<LedgerEntry>
     doPostgresSpecificOperation(soci::postgresql_session_backend* pg) override
     {
-        assert(mAccountIDs.size() == mIssuers.size());
-        assert(mAccountIDs.size() == mAssetCodes.size());
+        assert(mAccountIDs.size() == mAssets.size());
 
         std::string strAccountIDs;
-        std::string strIssuers;
-        std::string strAssetCodes;
+        std::string strAssets;
         marshalToPGArray(pg->conn_, strAccountIDs, mAccountIDs);
-        marshalToPGArray(pg->conn_, strIssuers, mIssuers);
-        marshalToPGArray(pg->conn_, strAssetCodes, mAssetCodes);
+        marshalToPGArray(pg->conn_, strAssets, mAssets);
 
         auto prep = mDb.getPreparedStatement(
             "WITH r AS (SELECT unnest(:v1::TEXT[]), "
-            "unnest(:v2::TEXT[]), "
-            "unnest(:v3::TEXT[])) SELECT accountid, assettype, "
-            "assetcode, "
-            "issuer, tlimit, balance, flags, lastmodified, "
-            "extension, ledgerext"
+            "unnest(:v2::TEXT[])) SELECT accountid, asset, "
+            "ledgerentry, lastmodified "
```

Comment:
I don't think this should be reading `lastmodified`.

Reply:
Good catch. Removed.